x-pack/test/functional/apps/ml/data_frame_analytics/outlier_detection_creation.ts

```diff
@@ -168,6 +168,13 @@ export default function ({ getService }: FtrProviderContext) {
             testData.createIndexPattern
           );
 
+          await ml.testExecution.logTestStep('continues to the validation step');
```

Comment:
Looking at the CI failure, you will also need to add this step into the accessibility test `x-pack/test/accessibility/apps/ml·ts` for the outlier job.

Reply:
Updated in 64b4e9dd5da186820cd28990b244d906bc3d3964